src/Components/ColorPalette/ColorPalette.jsx

```diff
@@ -0,0 +1,52 @@
+import { toBePartiallyChecked } from "@testing-library/jest-dom/dist/matchers";
+import { useState } from "react";
+import "./ColorPalette.css";
+
+export function ColorPalette({ noteCard, setNoteCard }) {
+  const colors = [
+    "#d1d1d1",
+    "#e1dbd6",
+    "#e2e2e2",
+    "#edf2fb",
+    "#d7e3fc",
+    "#ccdbfd",
+    "#FF0000",
+  ];
+  const [paletteDisplay, setPaletteDisplay] = useState(false);
+  return (
+    <div
+      className="palette-btn"
+      onMouseOver={() => {
+        setPaletteDisplay(!paletteDisplay);
+      }}
+      onMouseOut={() => {
+        setPaletteDisplay(!paletteDisplay);
+      }}
```

Comment:
This was new way for me to handle the view of the component when the mouse is placed over it